stock_quant_manual_assign/tests/test_stock_quant_manual_assign.py

```diff
@@ -149,7 +149,7 @@ def test_quant_manual_assign(self):
             "There are 2 quants selected",
         )
         self.assertFalse(self.move.picking_type_id.auto_fill_qty_done)
-        self.assertEqual(sum(self.move.move_line_ids.mapped("qty_done")), 0.0)
```

Comment:
because of "qty_done" was removed in 17.0, so I think we don't need these test case.